src/App.css

```diff
@@ -0,0 +1,15 @@
+.App {
+  text-align: center;
+}
+
+body {
+  background: #3AAFAB;
+  color: white;
+  font: normal 18px/1.5 "Fira Sans", "Helvetica Neue", sans-serif;
+}
+
+.story {
+  border: 1px solid white;
+  margin-left: 20px;
+  margin-right: 20px;
```

Comment:
Based on the way we've taught things, this ruleset should be in a separate file `components/Story.css`.